.vscode/settings.json

```diff
@@ -18,5 +18,11 @@
   },
   "jest.enableInlineErrorMessages": true,
   "jest.showCoverageOnLoad": true,
-  "jest.runAllTestsFirst": false
+  "jest.runAllTestsFirst": false,
+  "[typescript]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
CLI uses prittier for formatting. If we are setting this we should set prettier as formatter